### DISCUSSION TITLE
AlsStanceTags::Standing

### DISCUSSION THREAD

**Discussion by mlnant11** [Sixze/ALS-Refactored#456] [Q&A]
Hi! It's been a while I was studying the code of als and today I had a doubt.
Look the code below:

void UAlsAnimationInstance::PlayTransitionAnimation(UAnimSequenceBase* Animation, const float BlendInDuration, const float BlendOutDuration,
                                                    const float PlayRate, const float StartTime, const bool bFromStandingIdleOnly)
{
	if (bFromStandingIdleOnly && (LocomotionState.bMoving || Stance != AlsStanceTags::Standing))
	{
		return;
	}
	
	my doubt is about "AlsStanceTags::Standing".
	this tag need to have an value, correct? if its not have a value, how it would be compared?
	so, WHERE is this value tag being defined? its in blueprint? or its in code?
	this tag is being related to an animation?
	I'm sorry if the doubt is fool, I just started learning als a few weeks ago.
	please, can someone tell me exactly where is this being defined?

**Comment by ameaninglessname** [comment]
https://github.com/Sixze/ALS-Refactored/blob/3fde9f486bef84f612db52e7a7dc67d40a43e8c6/Source/ALS/Private/Utility/AlsGameplayTags.cpp#L24

[Rider could help you navigate the code](https://www.jetbrains.com/rider/nextversion/)

**Reply by mlnant11** [reply]
oh, thanks, helped a lot!
but now I have another question... the folder "AlsGameplayTags.cpp" does not appear on my editor.
all the content in the "private" folder just disappeared, thats why I coudnt find this answer before.

![image](https://github.com/Sixze/ALS-Refactored/assets/162394350/96676451-49e4-4286-bf91-43e53cb5cd8e)

look, only public folders!

**Reply by Magic-Summer** [reply]
@mlnant11 The `Private` folders are not visible in the UE Content Browser. The best place to see them is in your IDE. 
As @ameaninglessname mentioned, Rider is a great IDE, and free to use if you are a student or fall into a similar category.
